### ISSUE TITLE
Label for date selection popup button

### ISSUE THREAD

**Issue by smythp** [Bedework/bedework#139]
The button to open the date selection popup (in "manage events," the button that looks like a small calendar near "start date") is not labelled correctly. It appears as "elipsis" in screen readers and is designated as an image and not a button.
